functions/Import-DbaCsv.ps1

```diff
@@ -300,7 +302,7 @@ function Import-DbaCsv {
                 [System.Data.SqlClient.SqlTransaction]$transaction
             )
             $reader = New-Object LumenWorks.Framework.IO.Csv.CsvReader(
-                (New-Object System.IO.StreamReader($Path)),
+                (New-Object System.IO.StreamReader($Path, [System.Text.Encoding]::$Encoding)),
```

Comment:
I remember this problem now! It was hard as hell. I did not and would not have thought to move the encoding here, thank you so much 🙇‍♀️

Reply:
Still at a loss to explain why those constructors wouldn't work! 
My C# ain't good enough to figure it out :)

Reply:
mine either! i thought perhaps the docs were wrong but i used powershell to tell me the constructors and they said the same thing 🤔